cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -1235,7 +1233,9 @@ class SchedulerPluginRequirementsSchema(BaseSchema):
     )
     requires_sudo_privileges = fields.Bool(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
     supports_cluster_update = fields.Bool(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
-    supported_parallel_cluster_versions = fields.Str(metadata={"update_policy": UpdatePolicy.UNSUPPORTED})
+    supported_parallel_cluster_versions = fields.Str(
+        metadata={"update_policy": UpdatePolicy.UNSUPPORTED}, validate=validate.Regexp(r"[a-z0-9\>\=\<\.]")
```

Comment:
With (r"[a-z0-9\>\=\<\.]"), it is able to filter out "!" and match ">=3.0.0, <=3.1.0" with multiple matches. Changed to r"^[a-z0-9\>\=\<\.\s]*$" so it match the entire string instead of each char

Reply:
this regex is always true because you are asking to match 0 or more occurrences of the chars in []. What about something like this: `^((>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+,\s*)*(>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+$`? You can use tools like this to double check what it does: https://regex101.com/r/M0Fk1W/1

Reply:
Thank you! I see, with `^((>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+,\s*)*(>|<|>=|<=)?[0-9]+\.[0-9]+\.[0-9]+$`, we also make it not possible to filter out format like `<>3.0.0` to be invalid. I was only think about filter out string contains "!"